test/integration/targets/mysql_db/tasks/state_dump_import.yml

```diff
@@ -58,12 +73,52 @@
 - name: state dump/import - file name should exist
   file: name={{ db_file_name }} state=file
 
+- name: state dump with multiple databases in comma separated form.
+  mysql_db:
+    name: "{{ db_name }},{{ db_name2 }}"
+    state: dump
+    target: "{{ dump_file1 }}"
+    login_unix_socket: '{{ mysql_socket }}'
+  register: dump_result1
+
+- name: assert successful completion of dump operation (with multiple databases in comma separated form)
+  assert:
+    that:
+       - "dump_result1.changed == true"
+
+- name: state dump - dump file1 should exist
+  file: name={{ dump_file1 }} state=file
+
+- name: state dump with multiple databases in list form.
+  mysql_db:
+    name:
+      - "{{ db_name }}"
+      - "{{ db_name2 }}"
+    state: dump
+    target: "{{ dump_file2 }}"
+    login_unix_socket: '{{ mysql_socket }}'
+  register: dump_result2
+
```

Comment:
we should also check check_mode that it works with changed=False and nothing actually changed after the playbook execution, i.e. the dump files won't be created

Reply:
Added check_mode